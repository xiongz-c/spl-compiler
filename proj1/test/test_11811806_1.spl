int 0506var;
char myChar;;

int main(int _a, int 1_arg){
  char hex = '\x888';
  int leadingZero = -0099;
  float wrongFloat = 00.00;
  float validFloat = 0.0;
  int validInt = 0x0;
  return 0
}