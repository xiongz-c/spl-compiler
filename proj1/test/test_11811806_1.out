Error type A at Line 1: unknown lexeme 0506var
Error type B at Line 2: Multiple ';'
Error type A at Line 4: unknown lexeme 1_arg
Error type A at Line 5: unknown lexeme '\x888'
Error type A at Line 6: unknown lexeme -0099
Error type A at Line 7: unknown lexeme 00.00
Error type B at Line 10: Missing semicolon ';'
